app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int | float,
+                 clean_mark: int | float,
+                 brand: int | float) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
+
+car = Car(comfort_class=0, clean_mark=0, brand=0)
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int | float,
+                 clean_power: int | float,
+                 average_rating: int | float,
+                 count_of_ratings: int | float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, class_car_instance: Car) -> Any:
+        return (round(class_car_instance.comfort_class
+                * (self.clean_power - class_car_instance.clean_mark)
+                * self.average_rating / self.distance_from_city_center, 1))
+
+    def wash_single_car(self, class_car_instance: Car) -> Any:
+        if class_car_instance.clean_mark <= self.clean_power:
+            single_car_price = self.calculate_washing_price(class_car_instance)
+            class_car_instance.clean_mark = self.clean_power
+            return single_car_price
+
+    def serve_cars(self, cars_list: list) -> Any:
+        income = 0
+        for unit in cars_list:
+            if unit.clean_mark < self.clean_power:
+                income += self.wash_single_car(unit)
+                unit.clean_mark = self.clean_power
```

Comment:
In the `serve_cars` method, the `clean_mark` is set to `self.clean_power` twice for each car. This is redundant and can be optimized by removing the second assignment inside the loop.